        /** Mark that the exception has associated throws tag. */
        final void setFound() {
            found = true;
        }

